core/src/main/java/be/isach/ultracosmetics/cosmetics/morphs/MorphSheep.java

```diff
@@ -28,18 +27,16 @@ public void onLeftClick(PlayerInteractEvent event) {
         event.setCancelled(true);
         sound.play();
         SheepWatcher sheepWatcher = (SheepWatcher) getDisguise().getWatcher();
-        new BukkitRunnable() {
-            private int count = 0;
 
-            @Override
-            public void run() {
-                if (count > 9) {
-                    cancel();
-                    return;
-                }
-                sheepWatcher.setColor(DyeColor.values()[RANDOM.nextInt(DyeColor.values().length)]);
-                count++;
+        final int[] count = {0};
```

Comment:
Interesting workaround!

Reply:
Might be better to use an AtomicInteger here, I can refactor

Reply:
Done